sci-libs/dlib/files/dlib-19.24.8-disable-upstream-flags.patch

```diff
@@ -0,0 +1,111 @@
+diff --git a/dlib/test/CMakeLists.txt b/dlib/test/CMakeLists.txt
+index 330ac241..6a3ad681 100644
+--- a/dlib/test/CMakeLists.txt
++++ b/dlib/test/CMakeLists.txt
+@@ -168,56 +168,56 @@ add_executable(${target_name} main.cpp tester.cpp
+ get_filename_component(DLIB_FFMPEG_DATA ${CMAKE_SOURCE_DIR}/ffmpeg_data/details.cfg REALPATH)
+ target_compile_definitions(${target_name} PRIVATE DLIB_FFMPEG_DATA="${DLIB_FFMPEG_DATA}")
+ 
+-if (CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
We technically only need to patch this line and make the if statement fail unconditionally. This would make the patch a lot shorter.